static/app/stores/alertStore.tsx

```diff
@@ -109,6 +110,8 @@ const storeConfig: Reflux.StoreDefinition & Internals & AlertStoreInterface = {
   },
 };
 
-const AlertStore = Reflux.createStore(storeConfig) as Reflux.Store & AlertStoreInterface;
+const AlertStore = makeSafeRefluxStore(
+  Reflux.createStore(storeConfig) as Reflux.Store & Internals & AlertStoreInterface
+);
```

Comment:
If we add the `Internals` type here won't the exported store now have the internal methods associated to it?

Reply:
Like in this case I can now access public `alerts` and `count` properties?

Reply:
Ah yes, that was a mistake. I will remove that